lua/nvim-tree.lua

```diff
@@ -151,6 +151,36 @@ local keypress_funcs = {
     )
     luv.unref(process.handle)
   end,
+  trash = function(node)
```

Comment:
could you move this function in it's own file ? something like `trash.lua`, the main file is getting very messy